test/mcmc/abstractmcmc.jl

```diff
@@ -34,44 +34,91 @@ function initialize_nuts(model::Turing.Model)
 end
 
 
-function initialize_mh(model)
+function initialize_mh_rw(model)
     f = DynamicPPL.LogDensityFunction(model)
     d = LogDensityProblems.dimension(f)
     return AdvancedMH.RWMH(MvNormal(Zeros(d), 0.1 * I))
 end
 
+# TODO: Should this go somewhere else?
```

Comment:
Probably DynamicPPL I'd say?

Reply:
Was thinking the same. Only thing holding me back is that it's unclear to me if this is indeed how we want to implement something like this :confused: It was a quick--and-dirty way of doing it, hence why I figured I'd just leave it here for now.